Makefile

```diff
@@ -296,7 +297,11 @@ pyformat:
 bootloader_protos:
 	@echo "Compiling protos..."
 	@mkdir -p $(BOOTLOADER_DIR)/protogen
-	@protoc -I=$(BOOTLOADER_DIR)/protos --c_out=$(BOOTLOADER_DIR)/protogen $(BOOTLOADER_DIR)/protos/*.proto
+	@for i in $$(ls $(BOOTLOADER_DIR)/protos); do \
+		protoc -I=$(BOOTLOADER_DIR)/protos -ocommand.pb $$i; \
+		python $(PYTHONPATHNANO)/generator/nanopb_generator.py -I=$(BOOTLOADER_DIR)/protos command.pb; \
+	done
+	@mv *.pb *.pb.c *.pb.h $(BOOTLOADER_DIR)/protogen
```

Comment:
You should probably remove `command.pb` after this, I think it's an intermediate file?